### ISSUE TITLE
AOB wildcard implementation 

### ISSUE THREAD

**Issue by MernGG** [l4tt/pygg#4]
The aob scanner in memory.py takes in an aob that has the type bytes so you cant actually pass in wildcards since a question mark isnt a byte character. Probably should pass it in as a string, handle the wildcards, and convert to bytes.

**Comment by l4tt** [comment]
That's true, it does convert wildcards from ? to /0x0/ but since I passed the argument to only take bytes. Il fix it to take a string

**Comment by l4tt** [comment]
fixed go check commit history, and hook function has been fixed you will have to provide a byte string though, but it can contain x00, and ?? example

"/x74/x32/??/??"
or
"/x74/x32/x00/x00"
